netci.groovy

```diff
@@ -5,36 +5,60 @@ import jobs.generation.Utilities;
 def project = GithubProject
 def branch = GithubBranchName
 
-[true, false].each { isPR ->
-    ['bvt', 'functional'].each { testCategory ->
-        def newJobName = "${testCategory}"
-        def testScript = "Test.cmd";
-        if (testCategory == 'functional') { testScript = "TestAll.cmd" }
+def platformList = ['Linux', 'Windows_NT']
 
-        def newJob = job(Utilities.getFullJobName(project, newJobName, isPR)) {
+platformList.each { platform ->
+    if (platform == 'Linux') {
+        def os = 'Linux'
+        def architecture = 'x64'
+        def configuration = 'Release'
+        def isPR = true
```

Comment:
we should probably add the Linux job for both PR and non-PR

Reply:
Updated.